app/src/main/java/org/schabi/newpipe/ktx/TextView.kt

```diff
@@ -0,0 +1,48 @@
+@file:JvmName("TextViewUtils")
+
+package org.schabi.newpipe.ktx
+
+import android.animation.Animator
+import android.animation.AnimatorListenerAdapter
+import android.animation.ArgbEvaluator
+import android.animation.ValueAnimator
+import android.util.Log
+import android.widget.TextView
+import androidx.annotation.ColorInt
+import androidx.interpolator.view.animation.FastOutSlowInInterpolator
+import org.schabi.newpipe.BuildConfig
+
+private const val TAG = "TextViewUtils"
+private const val DEBUG = BuildConfig.BUILD_TYPE != "release"
```

Comment:
Please import the DEBUG const from MainActivity, so we only need to change one var when there is the need to change the debug logic.

Reply:
> Oh, then also `TAG` should be changed to `= CurrentJavaClass.class.getSimpleName()`

There doesn't seem to be a way to get a reference to the generated class for top-level declarations.